tools/AutoTuner/src/autotuner/distributed.py

```diff
@@ -397,10 +397,13 @@ def parse_config(config, path=os.getcwd()):
                 )
         # Default case is VAR=VALUE
         else:
+            # FIXME there is no robust way to get this metainformation from
+            # ORFS about the variables, so disable this code for now.
+
             # Sanity check: ignore all flow variables that are not tunable
-            if key not in flow_variables:
-                print(f"[ERROR TUN-0017] Variable {key} is not tunable.")
-                sys.exit(1)
+            # if key not in flow_variables:
+            #     print(f"[ERROR TUN-0017] Variable {key} is not tunable.")
+            #     sys.exit(1)
```

Comment:
How does this problem manifest?

Reply:
When we modify the scripts/*.tcl so that the parsing of the scripts/*.tcl variables no longer find all the variables such as CTS_CLUSTER_SIZE, then AutoTuner mistakenly things it isn't a tunable parameter.

What is needed is metainformation about the variables so that AutoTuner can check if a string matches a variable name, ideally some extra features can be added like legal values/range(boolean, float, integer, ranges, enumerations, etc.).